### PR TITLE
feat: optimize component styles

### PR DESCRIPTION
**Pull request by noneAuth**
### 👀 PR includes

✨ Feature

- [1 ] New feature
  update flex container auto & tools icon styles update
🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [ ] Solve the issue and close #0

🔧 Chore

- [ ] Test case
- [ ] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌     | ✅    |

### 🔍 Self-Check before the merge

- [ ] Add or update relevant docs.
- [ ] Add or update relevant demos.
- [ ] Add or update test case.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

File: packages/gi-assets-basic/src/components/RichContainer/Component.tsx [hunk 1]
```diff
@@ -27,7 +28,7 @@ export interface RichContainerState {
   viewMode: string;
 }
 const getDefaultSideWidth = () => {
-  const defaultWidth = localStorage.getItem('GI_RICH_CONTAINER_SIDE_WIDTH') || '330';
+  const defaultWidth = localStorage.getItem('GI_RICH_CONTAINER_SIDE_WIDTH') || '350';
```
Comment (lxfu1) [inline packages/gi-assets-basic/src/components/RichContainer/Component.tsx:31]: '350' => 350

File: packages/gi-assets-basic/src/components/RichContainer/Component.tsx [hunk 2]
```diff
@@ -58,7 +59,14 @@ const RichContainer = props => {
...
     if (isExpanded) {
       const defaultWidth = getDefaultSideWidth();
-      setWidth(defaultWidth >= MIN_WIDTH ? defaultWidth : MIN_WIDTH);
+      const checkAreaWidth = defaultWidth <= MAX_WIDTH && defaultWidth >= MIN_WIDTH;
+      if (checkAreaWidth) {
+        setWidth(defaultWidth);
+      } else if (defaultWidth >= MAX_WIDTH) {
+        setWidth(MAX_WIDTH);
+      } else if (defaultWidth <= MIN_WIDTH) {
+        setWidth(MIN_WIDTH);
```
Comment (lxfu1) [inline packages/gi-assets-basic/src/components/RichContainer/Component.tsx:68]:
```ts
const isInRange = defaultWidth <= MAX_WIDTH && defaultWidth >= MIN_WIDTH;
const width = (isInRange? defaultWidth : (defaultWidth >= MAX_WIDTH? MAX_WIDTH : MIN_WIDTH));
setWidth(width);
```

File: packages/gi-assets-basic/src/components/RichContainer/Component.tsx [hunk 3]
```diff
@@ -143,8 +158,10 @@ const RichContainer = props => {
       value: 'GISDK_CANVAS',
       label: (
         <Space>
-          <Icon type={ViewArea.icon}></Icon>
-          {$i18n.get({ id: 'basic.components.RichContainer.Component.GraphView', dm: '图谱视图' })}
+          <Icon type={ViewArea.icon} style={{ fontSize: '18px' }}></Icon>
```
Comment (lxfu1) [inline packages/gi-assets-basic/src/components/RichContainer/Component.tsx:161]: fontSize: 18   下同

Comment (noneAuth) [reply packages/gi-assets-basic/src/components/RichContainer/Component.tsx:161]: done

File: packages/gi-assets-basic/src/components/RichContainer/Component.tsx (hunk 1 above)
Comment (noneAuth) [reply packages/gi-assets-basic/src/components/RichContainer/Component.tsx:31]: done

File: packages/gi-assets-basic/src/components/RichContainer/Component.tsx (hunk 2 above)
Comment (noneAuth) [reply packages/gi-assets-basic/src/components/RichContainer/Component.tsx:68]: done

**Review by lxfu1 — APPROVED** [review]

**Pull request merged**